webapp/components/survey/TaxonomyDetails/EditHeader/EditHeader.js

```diff
@@ -15,15 +15,14 @@ import { useI18n } from '@webapp/store/system'
 import { useSurveyId } from '@webapp/store/survey'
 import { useAuthCanEditSurvey } from '@webapp/store/user'
 
-import * as TaxonomyActions from '../actions'
-import * as TaxonomyState from '../taxonomyState'
+import { State } from '../store'
 
-const TaxonomyEditHeader = () => {
-  const dispatch = useDispatch()
+const EditHeader = (props) => {
```

Comment:
why not just header? it can be readOnly for some user

Reply:
Agree!